java/code/src/com/redhat/rhn/domain/product/SUSEProductSCCRepository.java

```diff
@@ -57,6 +57,7 @@
     private String parentChannelLabel;
     private String channelName;
     private boolean mandatory;
+    private boolean installerUpdates = false;
```

Comment:
i think this makes more sense in the SCCRepository table

Reply:
Right. I moved it.